tests/test_codelists.py

```diff
@@ -34,3 +38,44 @@ def test_codelists_update(tmp_path, requests_mock):
     assert (codelist_dir / "project123-codelist456.csv").read_text() == "foo"
     assert not (codelist_dir / "project123-codelist789-version1.csv").exists()
     assert (codelist_dir / "project123-codelist098.csv").read_text() == "bar"
+    manifest = json.loads((codelist_dir / "codelists.json").read_text())
+    assert manifest["files"].keys() == {
+        "project123-codelist456.csv",
+        "project123-codelist098.csv",
+    }
+
+
+@pytest.fixture
+def codelists_path(tmp_path):
+    shutil.copytree(
+        Path(__file__).parent / "fixtures/codelists", tmp_path / "codelists"
```

Comment:
Good point. As it happens, I think Windows does the right thing with forward slashes but agree it would be better not to rely on this. This has also prompted me to check my belief that the tests were run on Windows in CI which isn't actually true, so I should fix that as well.